src/event/event.controller.ts

```diff
@@ -1,10 +1,23 @@
-import { Controller, Get, Body, Post, Param, HttpException, HttpStatus } from '@nestjs/common';
+import { Controller, Get, Body, Post, Param, HttpException, HttpStatus, UseInterceptors, UploadedFile } from '@nestjs/common';
+
+import { EventWithCreator, AttendeeWithUser } from "src/../prisma/types";
+import { CreateEventBody, FileResponseObject, ReactionResponseObject, RegisterAttendeeBody } from "src/types";
+
 import { EventService } from './event.service';
-import { EventWithCreator, CreateEventBody } from "../../prisma/types";
+import { FileService } from 'src/file/file.service';
+import { ReactionService } from 'src/reaction/reaction.service';
+import { AttendeeService } from 'src/attendee/attendee.service';
+
+import { FileInterceptor } from '@nestjs/platform-express/multer';
```

Comment:
here you are mixing up entities of your application and mixing them up on the same controller. you could improve this. 
event controller only should call event service or atteendes controller should only call atteendes service (and so on). 
if you want you can call a service from a service (example: events service calls atteendes service)